.github/workflows/yaml.yml

```
@@ -7,9 +7,13 @@ on: # yamllint disable-line rule:truthy
 
 jobs:
   lintyaml:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04-arm
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      # yamllint is not yet available in ubuntu-24.04-arm (https://github.com/actions/partner-runner-images/blob/main/images/arm-ubuntu-24-image.md)
+      # request https://github.com/actions/partner-runner-images/issues/56
+      - name: Install yamllint
+        run: sudo apt-get update && sudo apt-get install -y yamllint
```

consider adding `--no-install-recommends` to avoid installing any suggested packages
```suggestion
        run: sudo apt-get update && sudo apt-get install -y --no-install-recommends yamllint
```

---

fixed  ;)